### DISCUSSION TITLE
Sponsorblock seems to not work on https://youtu.be/LogQq9_-Y3I

### DISCUSSION THREAD

**Discussion by stdedos** [FreeTubeApp/FreeTube#5280] [General]
The introductory segment was not skipped, even after reloading and/or repeatedly pressing `0`.

... I know it's an issue, but I'm sure it's been reported somewhere already.
I'm trying to avoid the "you are opening too many duplicates" complain (even though **I did my part**, and https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+sponsorblock+skips, https://github.com/FreeTubeApp/FreeTube/issues?page=2&q=is%3Aissue+sponsorblock do not bring up open issues)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
duplicate of https://github.com/FreeTubeApp/FreeTube/issues/3534
